app/adapters/hathi/depositor.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+require "digest"
+require "zip"
+require "tmpdir"
+module Hathi
+  class Depositor
```

Comment:
Can we have a different class name here? `Depositor` to me suggests an individual.

Reply:
Also a small class-level comment describing what the class does would be good

Reply:
Renamed to Hathi::SubmisionInformationPackage, as per https://www.hathitrust.org/deposit_guidelines